test/BiochemistryAPI_server_test.py

```diff
@@ -182,26 +170,30 @@ def test_search_reactions(self):
         res = self.getImpl().search_reactions(
             self.ctx, {"query": "phosphohydrolase", "limit": 500}
         )[0]
-        self.assertEqual(len(res), 385)
+        self.assertEqual(len(res), 393)
         self.assertEqual(res[0]["id"], "rxn00001")
 
     def test_substructure_search(self):
```

Comment:
For future reference, another way you can skip tests is:
```
    @unittest.skip("demonstrating skipping")
    def test_nothing(self):
        self.fail("shouldn't happen")
```
[source](https://docs.python.org/3/library/unittest.html#skipping-tests-and-expected-failures)